packages/@uppy/core/src/index.test.js

```diff
@@ -326,16 +326,16 @@ describe('src/Core', () => {
   describe('preprocessors', () => {
     it('should add a preprocessor', () => {
       const core = new Core()
-      const preprocessor = () => {}
+      const preprocessor = () => { }
```

Comment:
That doesn't seem to be necessary.
```suggestion
      const preprocessor = () => {}
```

Reply:
Happening because of eslint formatting. Shouldn't matter.